### DISCUSSION TITLE
Install tutorial video!

### DISCUSSION THREAD

**Discussion by DrishalMAC2** [kubilus1/autoortho#176] [Show and tell]
Hey guys! i made an installation tutorial video which hopefully will help people install it! 

Link here: https://youtu.be/4ihxhhHLMvA

This is an amazing addon which has been needed for X-Plane for a very long while. good job to the developers!

**Comment by dpdurst** [comment]
Nice Video!

**Reply by DrishalMAC2** [reply]
Hope it helped!

**Comment by gabrielcassis** [comment]
At the ~[8:20] mark, we can see a log entry "WARNING... Websocket transport not available. Install simple-websocket for improved performance.". I also get that message despite having installed simple-websocket from python and enabled websocket in IIS. I can't find any documentation on how to configure AutoOrtho to use websocket. Do you know how to?

**Reply by dpdurst** [reply]
That error you can actually ignore, I've got as well, read somewhere here or someplace else that it means nothing and you can safely ignore it.
